src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
             WriteProperty(jsonWriter, "indexName", searcher.Manager.IndexName);
+            WriteProperty(jsonWriter, "machineName", searcher.MachineName);
             WriteProperty(jsonWriter, "lastReopen", searcher.LastReopen);
+            WriteProperty(jsonWriter, "lastIndexReloadTime", searcher.LastIndexReloadTime);
+            WriteProperty(jsonWriter, "lastIndexReloadDurationInMilliseconds", searcher.LastIndexReloadDurationInMilliseconds);
```

Comment:
We should have a clear seperation between reporting status (like last reload time, last successful reload time), and reporting metrics (like reload speed, CPU utilization). The first need to be monitored by a job, and exposed directly by the service. The second should be reported to Geneva or AI, and monitored as metrics (like we currently do for CPU). 
The fact that MDM jobs can also expose metrics, shouldn't be a path to get our metrics into Geneva, we should use a more direct method to get them there.